### PR TITLE
Fix normal map palette

### LINKED ISSUE #3015: Normal map color wheel not working as intended
**Problem:**
When selecting a color with the color picker on the image of a normal map sphere, produced outside of Aseprite, the colors selected do not match the one displayed on the color picker. The color selected on the color wheel seem to not match the desired orientation when tested in Unity with default 3D lights.
I tried to use the normal color wheel to select colors between 2 already known and working colors, but it doesn't work at all.

**Steps to reproduce:** 
- import a normal map sphere (the one from the wikipedia page for example) in aseprite
- select a color with the color picker, preferably not at the center
- with the normal map palette selected, try to select the same color by clicking at the spot highlighted with a small circle on the color wheel.
- The new color is different from the previous one.

This capture shows the difference between the 2 colors, the first one (in the red circle), actually on the image, and the second, produced with the color wheel:
https://gyazo.com/e115b949781c6c0b77a3ca19bfe563fb

As per my tests, the color produced with Aseprite normal color wheel **are not correct** when the texture is imported in a program to light them.

As an added request, while testing some other programs to edit normal map I found one that shows a wireframe 3D square around the color selected on the wheel, clearly giving the information of the direction of the normal. That would be very helpful to wrap your head around what a color will really do when lighted.

If you need any feedback or to test fixes, don't hesitate to contact me.

### Aseprite and System version

* Aseprite version: 1.2.29-x64, steam version
* System: W10

### PR DESCRIPTION
**Pull request by cs-altshift**
Closes #3015 

I agree that my contributions are licensed under the Individual Contributor License Agreement V3.0 ("CLA") as stated in https://github.com/aseprite/sourcecode/blob/main/cla.md

I have signed the CLA following the steps given in https://github.com/aseprite/sourcecode/blob/main/sign-cla.md#sign-the-cla

### CODE REVIEW

**Comment by Gasparoken** [comment]
Thank you @cs-altshift! I reviewed your contribution.

Taking your commit as base, I'll add a few steps more to the Discrete mode and I'll replace the if-elseif-else condition by a couple of formulas.

File: src/app/ui/color_wheel.cpp [hunk 1]
```diff
@@ -102,32 +102,63 @@ app::Color ColorWheel::getMainAreaColor(const int _u, const int umax,
...
+          blueAngleDegrees = 45;
+        }
+        else if (normalizedDistance < 0.75) {
+          normalizedDistance = 0.5;
+          blueAngleDegrees = 30;
+        }
+        else {
+          normalizedDistance = 0.75;
+          blueAngleDegrees = 15;
+        }
```
[the remark this answers is not in the document]
Comment (Gasparoken) [reply src/app/ui/color_wheel.cpp:145]: This calculates the discrete mode along 6 steps (along the radius), and simplifies the `if-elseif-else` statement.

**Review by Gasparoken — COMMENTED** [review]
I'll talk with David prior to merge this fix. Surely, we'll change the `if` block and some minor details. Thank you again!🎉

**Comment by cs-altshift** [comment]
Great news everything works as expected for you. AFAIK we do not use the discrete mode so I just matched what existed before. But indeed more steps will be better.

**Comment by dacap** [comment]
Sorry @cs-altshift that we weren't able to merge this PR yet, I'm preparing a corporate CLA and probably changing the CLA-signing process soon. It will take some time until we can merge this but we'll keep you informed.

**Comment by cs-altshift** [comment]
> Sorry @cs-altshift that we weren't able to merge this PR yet, I'm preparing a corporate CLA and probably changing the CLA-signing process soon. It will take some time until we can merge this but we'll keep you informed.

No problem. We have a working build on our side anyway.

**Comment by cs-altshift** [comment]
> Finally merge in [20902e3](https://github.com/aseprite/aseprite/commit/20902e35b8f493daa65d90c9ef2fd07849bad7e3), thanks a lot @cs-altshift 🙏

You're welcome. Glad we will finally be able to use the mainstream Aseprite soon!

**Pull request closed without merging**